CONTRIBUTING.md

```diff
@@ -45,6 +54,15 @@ Everything within this section is the definitions for the actual PR policy follo
 
 **Final Comment Period**: 3 days
 
+### `T: documentation`
```

Comment:
Instead of having two triage tags that mean effectively the same thing, I'd recommend renaming `T: refactor` to `T: small change` or something similar.

Reply:
The thing is that T: documentation is 3 days while T: refactor is 5; Maybe `t: minor` for docs and smaller refactors and `t: moderate` for the usual kind of refactors would work? I'll have to think about an ideal way of doing this

Reply:
Huh? `t: refactor` is still 3 in this PR.

Reply:
...*facepalms* i looked at `t: new apí`